runtime/help/options.md

```diff
@@ -172,6 +172,10 @@ Here are the available options:
     default value: `unknown`. This will be automatically overridden depending
     on the file you open.
 
+* `helpsplit`: sets the split type to be used by the `help` command.
```

Comment:
Maybe make this a bit clearer? For example, keep this sentence as is, but also add:

```
Possible values:
 * `vsplit`: open help in a vertical split pane
 * `hsplit`: open help in a horizontal split pane
```

Reply:
Also, mention `helpsplit` in the documentation of the `help` command?